node/src/actors/json_rpc/json_rpc_methods.rs

```diff
@@ -700,21 +699,27 @@ pub fn send_value(params: Result<BuildVtt, jsonrpc_core::Error>) -> JsonRpcResul
     }
 }
 
-/// Node status
-#[derive(Debug, Default, Serialize)]
-#[serde(rename_all = "camelCase")]
-pub struct Status {
-    chain_beacon: CheckpointBeacon,
-    synchronized: bool,
-    num_peers_inbound: u32,
-    num_peers_outbound: u32,
-    num_active_identities: u32,
-    total_active_reputation: u64,
+/// Node synchronization status
+#[derive(Debug, Default, Deserialize, Serialize)]
+pub struct SyncStatus {
+    /// The hash of the top consolidated block and the epoch of that block
+    pub chain_beacon: CheckpointBeacon,
+    /// The current epoch
+    pub current_epoch: u32,
+    /// Num peers inbound
+    // TODO: remove? we have getPeers
+    pub num_peers_inbound: u32,
+    /// Num peers outbound
+    // TODO: remove? we have getPeers
```

Comment:
agree, it would have to be in only one CLI method, if we would include here I think we should remove the other method. But I prefer different methods